frontend/src/pages/MapView/SearchBar/index.jsx

```diff
@@ -65,9 +68,21 @@ const SearchBar = () => {
             Go
           </button>
         </div>
+        {tags.length > 0 && (<div className="card-tags">  
+          {tags.map((tag, idx) => (
+            <div className="card-tag" key={idx}>
```

Comment:
Small nit: Using `idx` as `key` [can cause unexpected behavior](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). Can use just the tag itself as a string.